<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileEditorUI</class>
 <widget class="QWidget" name="ProfileEditorUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>357</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="profileGroup">
     <property name="title">
      <string>Profile</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="profileNameLabel">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="profileIN"/>
        </item>
        <item>
         <widget class="QCheckBox" name="autoConnectCheck">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;After establishing connection with INDI server, automatically connect all devices.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Auto Connect</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="portSelectorCheck">
          <property name="toolTip">
           <string>Show port selector dialog when first connecting to equipment. It is automatically disabled after connection is established.</string>
          </property>
          <property name="text">
           <string>Port Selector</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="loadSiteCheck">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
&lt;p&gt;Send current KStars geographical site settings to drivers once Ekos is online.&lt;/p&gt;
&lt;p&gt;This option should only be used when connecting to a remote geographic site.&lt;/p&gt;
&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Site Info</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="guidingTypeLabel">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;body&gt;&lt;p&gt;Select which plug-in the Guider module should use.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Guiding:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="modeLabel">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
&lt;p&gt;Connection mode to use for the INDI server:&lt;/p&gt;
&lt;p&gt;&lt;span style=&quot;font-weight:600&quot;&gt;Local&lt;/span&gt;&lt;br/&gt;Connect to a local INDI server, created when Ekos starts.&lt;/p&gt;
&lt;p&gt;&lt;span style=&quot;font-weight:600&quot;&gt;Remote&lt;/span&gt;&lt;br/&gt;Connect to a remote INDI server that is already running.&lt;/p&gt;
&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Mode:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="9">
         <widget class="QLabel" name="INDIWebManagerPortLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Port:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="9">
         <widget class="QLabel" name="externalGuidePortLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Port:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="remoteMode">
          <property name="text">
           <string>Re&amp;mote</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="remoteHostLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Host:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="9">
         <widget class="QLabel" name="remotePortLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Port:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="10">
         <widget class="QLineEdit" name="INDIWebManagerPort">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;body&gt;&lt;p&gt;INDI Web Manager port.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>8624</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4" colspan="5">
         <widget class="QLineEdit" name="remoteHost">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;body&gt;&lt;p&gt;Remote INDI server host or address.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string notr="true">localhost</string>
          </property>
         </widget>
        </item>
        <item row="0" column="10">
         <widget class="QLineEdit" name="remotePort">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;body&gt;&lt;p&gt;Remote INDI Server Port.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>7624</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4" colspan="5">
         <widget class="QLineEdit" name="externalGuideHost">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;body&gt;&lt;p&gt;External guider host or address.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>localhost</string>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <widget class="QPushButton" name="indihubB">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Configure INDIHub.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>INDI Hub</string>
          </property>
          <property name="icon">
           <iconset resource="../data/kstars.qrc">
            <normaloff>:/icons/indihub.svg</normaloff>:/icons/indihub.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="localMode">
          <property name="text">
           <string>&amp;Local</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="10">
         <widget class="QLineEdit" name="externalGuidePort">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;body&gt;&lt;p&gt;External guider port.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="8">
         <widget class="QPushButton" name="scanB">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;body&gt;&lt;p&gt;Scan local network for INDI web managers.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Scan</string>
          </property>
          <property name="icon">
           <iconset theme="search">
            <normaloff>.</normaloff>.</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="externalGuideHostLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Host:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QComboBox" name="guideTypeCombo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Internal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PHD2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LinGuider</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0" colspan="4">
         <widget class="QCheckBox" name="INDIWebManagerCheck">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;body&gt;&lt;p&gt;Store profile on remote INDI Web Manager. Use INDI Web Manager on the remote device to start/stop INDI server.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>INDI Web Manager</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4" colspan="3">
         <widget class="QPushButton" name="openWebManagerB">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;body&gt;&lt;p&gt;Open Web Manager in browser.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Web Manager</string>
          </property>
          <property name="icon">
           <iconset theme="applications-internet">
            <normaloff>.</normaloff>.</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="driversGroupBox">
     <property name="title">
      <string>Select Devices</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="4">
       <widget class="QLabel" name="aux3Label">
        <property name="toolTip">
         <string>Auxiliary #3</string>
        </property>
        <property name="text">
         <string>Aux 3:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="aux4Label">
        <property name="toolTip">
         <string>Auxiliary #4</string>
        </property>
        <property name="text">
         <string>Aux 4:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Guider:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="aux1Label">
        <property name="toolTip">
         <string>Auxiliary #1</string>
        </property>
        <property name="text">
         <string>Aux 1:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="remoteDriversLabel">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add remote INDI drivers to chain with the local INDI server configured by this profile. Format this field as a comma-separated list of quoted driver name, host name/address and optional port:&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;ZWO ASI120MC&amp;quot;@192.168.1.50:8000&lt;/span&gt;&lt;br/&gt;Connect to the named camera on 192.168.1.50, port 8000.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;EQMod Mount&amp;quot;@192.168.1.50&lt;/span&gt;&lt;br/&gt;Connect to the named mount on 192.168.1.50, port 7624.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;@192.168.1.50:8000&lt;/span&gt;&lt;br/&gt;Connect to all drivers found on 192.168.1.50, port 8000.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;@192.168.1.50&lt;/span&gt;&lt;br/&gt;Connect to all drivers found on 192.168.1.50, port 7624.&lt;/p&gt;&lt;p&gt;When omitted, host defaults to localhost and port defaults to 7624.&lt;br/&gt;Remote INDI drivers must be already running for the connection to succeed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Remote:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="guiderCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="domeLabel">
        <property name="toolTip">
         <string>Dome Controller</string>
        </property>
        <property name="text">
         <string>Dome:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QComboBox" name="aux4Combo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="ccdCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="AOLabel">
        <property name="toolTip">
         <string>Adaptive Optics</string>
        </property>
        <property name="text">
         <string>AO:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QComboBox" name="aux3Combo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="weatherCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="aux2Label">
        <property name="toolTip">
         <string>Auxiliary #2</string>
        </property>
        <property name="text">
         <string>Aux 2:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="domeCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QComboBox" name="aux1Combo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QComboBox" name="aux2Combo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>CCD:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="AOCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="filterLabel">
        <property name="toolTip">
         <string>Filter Wheel</string>
        </property>
        <property name="text">
         <string>Filter:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Focuser:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Mount:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="focuserCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="filterCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="weatherLabel">
        <property name="toolTip">
         <string>Weather Station</string>
        </property>
        <property name="text">
         <string>Weather:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="mountCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="4">
       <widget class="QLineEdit" name="remoteDrivers">
        <property name="placeholderText">
         <string>driver@host:port,driver@host,@host:port,@host,driver</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QPushButton" name="scriptsB">
        <property name="text">
         <string>Scripts</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>profileIN</tabstop>
  <tabstop>localMode</tabstop>
  <tabstop>remoteHost</tabstop>
  <tabstop>remotePort</tabstop>
  <tabstop>INDIWebManagerCheck</tabstop>
  <tabstop>INDIWebManagerPort</tabstop>
  <tabstop>mountCombo</tabstop>
  <tabstop>ccdCombo</tabstop>
  <tabstop>guiderCombo</tabstop>
  <tabstop>focuserCombo</tabstop>
  <tabstop>filterCombo</tabstop>
  <tabstop>AOCombo</tabstop>
  <tabstop>domeCombo</tabstop>
  <tabstop>weatherCombo</tabstop>
  <tabstop>aux1Combo</tabstop>
  <tabstop>aux2Combo</tabstop>
  <tabstop>aux3Combo</tabstop>
  <tabstop>aux4Combo</tabstop>
  <tabstop>loadSiteCheck</tabstop>
 </tabstops>
 <resources>
  <include location="../data/kstars.qrc"/>
 </resources>
 <connections/>
</ui>
